### DISCUSSION TITLE
calc 2.15.1.0 release

### DISCUSSION THREAD

**Discussion by lcn2** [lcn2/calc#154] [Announcements]
The following are the changes in this release:

    Converted all ASCII tabs to ASCII spaces using a 8 character
    tab stop, for all files, except for all Makefiles (plus rpm.mk).
    The command `git diff -w` reports no changes.  There is no
    functionality change in calc: only ASCII tabs to ASCII spaces.

    Fixed trailblank.  It was pruning . in its find search.
    Added check for ASCII tabs is non-Makefiles.

    This version will form the basis for the calc v2 to calc v3 fork.

<hr /><em>This discussion was created from the release <a href='https://github.com/lcn2/calc/releases/tag/v2.15.1.0'>calc 2.15.1.0 release</a>.</em>

**Comment by pmetzger** [comment]
I updated MacPorts.

**Reply by lcn2** [reply]
Thank you 🙏 for this information and your support of calc under Macports, @pmetzger
